### ISSUE TITLE
Error-like graphic appears despite successful DDL statement

### ISSUE THREAD

**Issue by wangandi** [MaterializeInc/metabase-materialize-driver#9]
Just a usability issue.

If you run a DDL statement through the `Native Query` interface in Metabase, such as this:
```
CREATE MATERIALIZED VIEW orderline_count AS
SELECT count(*) FROM src_orderline;
```

The command succeeds, but you get this image:

<img width="394" alt="Screen Shot 2020-02-05 at 3 40 51 PM" src="https://user-images.githubusercontent.com/5522362/74265536-b7df6180-4cd0-11ea-8c12-86172d2b7314.png">

Arguably the "No results were returned by the query" is correct, but the big red X make it look like something has gone wrong even though no results is the expected result.

**Comment by bobbyiliev** [comment]
I'm currently following up on all of the issues raised for the old version of the driver.

Metabase is primarily designed as a tool for querying data and creating visualizations and dashboards, not for directly altering database schema using DDL statements.

I've verified that this has been changed to a less intimidating error:

<img width="760" alt="image" src="https://github.com/MaterializeInc/metabase-materialize-driver/assets/21223421/eea7aa38-a4a9-4f03-a006-7fc7eaeb41ef">

This is also not explicitly related to the driver, but is also valid for the Postgres driver along with a Postgres database and not necessarily Materialize.
